api_app/api/routes/workspaces.py

```diff
@@ -118,18 +117,23 @@ async def create_workspace(workspace_create: WorkspaceInCreate, response: Respon
 
 
 @workspaces_core_router.patch("/workspaces/{workspace_id}", status_code=status.HTTP_202_ACCEPTED, response_model=OperationInResponse, name=strings.API_UPDATE_WORKSPACE, dependencies=[Depends(get_current_admin_user)])
-async def patch_workspace(workspace_patch: ResourcePatch, response: Response, user=Depends(get_current_admin_user), workspace=Depends(get_workspace_by_id_from_path), workspace_repo: WorkspaceRepository = Depends(get_repository(WorkspaceRepository)), resource_template_repo=Depends(get_repository(ResourceTemplateRepository)), operations_repo=Depends(get_repository(OperationRepository)), resource_history_repo=Depends(get_repository(ResourceHistoryRepository)), etag: str = Header(...), force_version_update: bool = False) -> OperationInResponse:
+async def patch_workspace(resource_patch: ResourcePatch, response: Response, user=Depends(get_current_admin_user), workspace=Depends(get_workspace_by_id_from_path), workspace_repo: WorkspaceRepository = Depends(get_repository(WorkspaceRepository)), resource_template_repo=Depends(get_repository(ResourceTemplateRepository)), operations_repo=Depends(get_repository(OperationRepository)), resource_history_repo=Depends(get_repository(ResourceHistoryRepository)), etag: str = Header(...), force_version_update: bool = False) -> OperationInResponse:
     try:
-        patched_workspace, resource_template = await workspace_repo.patch_workspace(workspace, workspace_patch, etag, resource_template_repo, resource_history_repo, user, force_version_update)
+        cascade_enabled = not resource_patch.isEnabled
+        if cascade_enabled:
```

Comment:
I think cascade enabled is a bit misleading.
When I read it I thought that the user or someone decides whether or not they enable cascade.

The more acccurate terminology at least according to the code is 'should_perform_cascade' or 'should_propogate' 'is_cascade/propogate' something like this. Because it's not 'enablement' it's more about should you propagate the action

Reply:
I agree. Changed. Let me know what you think